src/views/contributors.ts

```diff
@@ -140,10 +152,13 @@ export class ContributorsView<
 
     const disableEditButton = !can.editMetadata
 
-    const { isCorresponding, email } = displayAttr
+    const { isCorresponding, email, id } = displayAttr
 
     if (!disableEditButton) {
-      container.addEventListener('click', this.handleClick)
+      container.addEventListener('click', () => {
+        const dataTracked = attrs.dataTracked as TrackedAttrs[]
```

Comment:
the `ContextMenu` will not be closed always, so to guarantee it's closed. select the node:
```
const { state, dispatch } = this.view
        dispatch(
          state.tr.setSelection(NodeSelection.create(state.doc, this.getPos()))
        )
```

and add override this method:
```
  public deselectNode = () => {
    this.props.popper.destroy()
  }
```